timed/employment/models.py

```diff
@@ -402,3 +402,15 @@ def calculate_worktime(self, start, end):
         balance = sum([balance[2] for balance in balances], timedelta())
 
         return (reported, expected, balance)
+
+    def get_active_employment(self):
+        """Get current employment of the user.
+
+        Get current active employment of the user.
+        If the user doesn't have a return None.
+        """
+        try:
+            current_employment = Employment.objects.get_at(user=self, date=date.today())
```

Comment:
(Not something to fix in this PR, but...) This will break for users west of UTC (such as America, ...). While the server might think that an employment has ended on a given day, the day may not be over in the user's timezone. Similarly, it will break for users east of UTC at the beginning of their employment. They may be at date X (where the contract started), but at UTC, it's still date X-1 and the contract is not yet active.

This can only be fixed by storing the user's timezone in the user model. Which however is something for another time.